internal/olm/operator/registry/configmap/deployment_test.go

```diff
@@ -0,0 +1,266 @@
+	})
+
+	Describe("withConfigMapVolume", func() {
+		It("should apply f to dep's pod template spec and apply volumes", func() {
+			dep := &appsv1.Deployment{
+				Spec: appsv1.DeploymentSpec{
+					Replicas: nil,
+					Template: corev1.PodTemplateSpec{
+						Spec: corev1.PodSpec{
+							NodeName: "testNode",
+						},
+					},
+				},
+			}
+			expecteddep := &appsv1.Deployment{
+				Spec: appsv1.DeploymentSpec{
+					Replicas: nil,
+					Template: corev1.PodTemplateSpec{
+						Spec: corev1.PodSpec{
+							NodeName: "testNode2",
+						},
+					},
+				},
+			}
+			volume := corev1.Volume{
+				Name: "testVolName",
+				VolumeSource: corev1.VolumeSource{
+					ConfigMap: &corev1.ConfigMapVolumeSource{
+						LocalObjectReference: corev1.LocalObjectReference{
+							Name: "testCmName",
+						},
+					},
+				},
+			}
+			expecteddep.Spec.Template.Spec.Volumes = append(expecteddep.Spec.Template.Spec.Volumes, volume)
+			f := withConfigMapVolume("testVolName", "testCmName")
+			f(dep)
+
+			Expect(dep.Spec.Template.Spec.Volumes).Should(Equal(expecteddep.Spec.Template.Spec.Volumes))
+		})
+	})
+
+	Describe("withContainerVolumeMounts", func() {
+		It("should apply f to dep's pod template spec and apply volumemounts", func() {
+			paths := []string{"testPath1", "testPath2"}
+			dep := &appsv1.Deployment{
+				Spec: appsv1.DeploymentSpec{
+					Replicas: nil,
+					Template: corev1.PodTemplateSpec{
+						Spec: corev1.PodSpec{
+							NodeName: "testNode",
+							Containers: []corev1.Container{
+								corev1.Container{},
+								corev1.Container{},
+							},
+						},
+					},
+				},
+			}
+			expecteddep := &appsv1.Deployment{
+				Spec: appsv1.DeploymentSpec{
+					Replicas: nil,
+					Template: corev1.PodTemplateSpec{
+						Spec: corev1.PodSpec{
+							NodeName: "testNode2",
+							Containers: []corev1.Container{
+								corev1.Container{},
+								corev1.Container{},
+							},
+						},
+					},
+				},
+			}
+			volumeMounts := []corev1.VolumeMount{}
+			for _, p := range paths {
+				volumeMounts = append(volumeMounts, corev1.VolumeMount{
+					Name:      "testVolName",
+					MountPath: p,
+				})
+			}
+			for i := range expecteddep.Spec.Template.Spec.Containers {
+				expecteddep.Spec.Template.Spec.Containers[i].VolumeMounts = append(expecteddep.Spec.Template.Spec.Containers[i].VolumeMounts, volumeMounts...)
+			}
+			f := withContainerVolumeMounts("testVolName", paths...)
+			f(dep)
+
+			Expect(dep.Spec.Template.Spec.Containers).Should(Equal(expecteddep.Spec.Template.Spec.Containers))
+		})
+	})
+
+	Describe("getDBContainerCmd", func() {
+		It("should apply f to dep's pod template spec and apply volumes", func() {
+			initCmd := fmt.Sprintf("/bin/initializer -o %s -m %s", registryDBName, containerManifestsDir)
+			srvCmd := fmt.Sprintf("/bin/registry-server -d %s -t %s", registryDBName, registryLogFile)
+
+			Expect(getDBContainerCmd(registryDBName, registryLogFile)).Should(Equal(fmt.Sprintf("%s && %s", initCmd, srvCmd)))
```

Comment:
I would actually remove these 4 lines. It occurred to me that the one I gave you basically duplicates this test.